### PR TITLE
Added tokenize_function

### PR DESCRIPTION
**Pull request by Viandya**
added tokenize_function, removed some comments and uploaded some changes to pretrain.py file.

### CODE REVIEW

**Comment by devium335** [comment]
Why is it needed to redefine the token as the function to training? The token function also returns the token the same way as defined here.

**Review by devium335 — CHANGES_REQUESTED** [review]
This seems to add unnecessary code

File: pretrain.py [hunk 1]
```diff
@@ -15,29 +16,37 @@
...
-### TODO: Init the config and the model
+# Load the pretrained tokenizer
+tokenizer = PreTrainedTokenizerFast(
+    tokenizer_file=...,
+    mask_token="[MASK]",
+    pad_token="[PAD]",
+    cls_token="[CLS]",
+    sep_token="[SEP]",
+    unk_token="[UNK]",
+)
```
Comment (devium335) [inline pretrain.py:42]: The tokenizer is defined earlier

File: pretrain.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import argparse
 
 from datasets import load_dataset
+from transformers import PreTrainedTokenizerFast
```
Comment (devium335) [inline pretrain.py:4]: Unnecessary as this shouldn't be trained in this file

File: pretrain.py [hunk 3]
```diff
@@ -15,29 +16,37 @@
...
 dataset = load_dataset(config.data)
 
+# Train the tokenizer
+tokenizer = train_tokenizer(dataset=dataset, tokenization_type=config.tokenizer, vocab_size=config.vocab_size)
+
 
-### Train the tokenizer & Load the pretrained tokenizer
-# ! Multiple Possible Tokenizers
-# TODO: Find the best tokenizer
+def tokenize_function(examples):
```
Comment (devium335) [inline pretrain.py:26]: What is examples?

**Review by devium335 — CHANGES_REQUESTED** [review]

**Comment by Viandya** [comment]
oh, apparently I submitted the wrong code to the pull request, sorry, this code is really useless

**Pull request closed without merging**